source/agora/consensus/EnrollmentManager.d

```diff
@@ -544,6 +559,26 @@ public class EnrollmentManager
         return enrolls;
     }
 
+    /***************************************************************************
+
+        Get a pre-image for revelation
+
+        Params:
+            preimage = will contain the PreimageInfo if exists
+
+        Returns:
+            true if the pre-image exists
+
+    ***************************************************************************/
+
+    public bool getNextPreimage (out PreimageInfo preimage) @safe
```

Comment:
Where are the unittests for this function?

Reply:
I missed it. I will add it in a related PR.